### DISCUSSION TITLE
How do I find all entries with autotype set up?

### DISCUSSION THREAD

**Discussion by phirestalker** [keepassxreboot/keepassxc#10310] [Q&A] [answered]
I am moving to another password manager, but import and export for these things doesn't get it all. I need to find all the entries that need special attention to move the data over. Many of my entries have custom auto-type entries set up. How can I filter or search for these entries?

**Comment by droidmonkey** [comment, marked as answer]
You cannot, but you can use the keepassxc-cli to export your database to xml and search the xml. What other password manager uses the keepass auto type sequences?

**Reply by phirestalker** [reply]
I'm not sure. The one I switched to sure doesn't. I am going to reuse them however and write my own tool to do the autotyping. Yes, the other features and stability of this other password manager are that good (it is vaultwarden).

**Reply by phirestalker** [reply]
That worked great, and I found all the autotype fields. I have noticed that the custom fields are only a string type and not marked as custom in any other way. Do you have any tips on how to find the custom fields (in KeepassXC or in the XML)?

**Reply by droidmonkey** [reply]
There are a list of default attributes, everything else is "custom": 

https://github.com/keepassxreboot/keepassxc/blob/develop/src%2Fcore%2FEntryAttributes.cpp#L25-L29
